### ISSUE TITLE
Темы открытых столов

### ISSUE THREAD

**Issue by ai** [spb-frontend/talks#3]
Напишите несколько тем, о которых вы хотели бы пообщаться на след. встрече.

На следующей встрече мы сделаем тематические столы, желающие будут собираться вокруг флажка с темой и холливарить. Вначале нам нужно определиться с темами, которые наиболее всем интересны — потом мы найдём ведущих.

**Comment by ai** [comment]
Сборка CSS, ES6, io.js

**Comment by Zmoki** [comment]
gulp, тестирование, оптимизация и ускорение, io.js

**Comment by Realetive** [comment]
Style guide generator

**Comment by vforvova** [comment]
Сборка CSS, JS, gulp'ом из bower пакетов

**Comment by slowkot** [comment]
es6, gulp/grunt/bower, something new

**Comment by marinintim** [comment]
ES6

**Comment by Akurganow** [comment]
Тестирование интерфейсов (a/b и т.п), сборка, чё там нового))

**Comment by goq** [comment]
grunt/gulp/npm postcss

**Comment by Realetive** [comment]
Да, кстати, поддерживаю @Akurganow — gemini кто-нибудь юзал? Не a/b, конечно, но тоже из этой грядки.

**Comment by maksimr** [comment]
Обсудить проекты на angular:
- Архитектура
- структура
- e2e тестирование
- Инфраструктура(webpack, grunt, gulp)

**Comment by vovanbo** [comment]
ReactJS, Polymer, Web Components

**Comment by ai** [comment]
Отлично, с темами мы определились. Теперь давай-те разбирать столы — кто будет ведущим какого стола?

Я могу взять сборку CSS и JS — ES6, PostCSS, Gulp.

**Comment by maksimr** [comment]
Могу взять стол с *тестированием*

**Comment by Termina1** [comment]
Предлагаю следующие столы (на базе вышеперечисленного):
1) Сборка (ведет @ai)
2) Веб-компоненты и фреймворки (ведет Андрей Складников, докладчик)
3) Тестирование (unit, integration и т.д.) (ведет @maksimr)
4) ES6, ES7 и грядущие Web API (нужен ведущий)

Как вариант, можно оставить так?

**Comment by Jabher** [comment]
Извините, задержался :)
По фреймворкам - уже занято, как я вижу, могу взять ES6, 7, препроцессоры, будущие фичи браузеров.

**Comment by ai** [comment]
Отлично, давай все «ES6, 7, препроцессоры, будущие фичи браузеров» вместе

**Comment by Termina1** [comment]
@ai что-то через чур жирно будет для одного стола

**Comment by ai** [comment]
Ну логика у всех этим тем есть — будущее JS. Препроцессоры JS же имеются в виду?

**Comment by Termina1** [comment]
А, так отдельный стол про сборку будет? Я просто решил, ты их хочешь объеденить.

**Comment by ai** [comment]
Компиляция js и сборка ассетов немного разные вещи. Тем более у меня будет больше в сторону CSS.

**Comment by Jabher** [comment]
@Termina1 Я имел ввиду срач "Traceur VS 6to5" и тому подобные.
compile-to-JS языки я трогать не буду, конечно
